src/entities/user.ts

```diff
@@ -0,0 +1,22 @@
+// The user entity
```

Comment:
This entity gets created by the API enrichment after looking up the JWT.  So we won't need it in the tracker.

Reply:
ok, cool